package.json

```diff
@@ -13,10 +13,13 @@
     "write-translations": "docusaurus write-translations",
```

Comment:
Can you run `npm run prettier:fix` and apply it as a single commit here?
It would make sense to have it close to the actual prettier config. Thanks!

And also add:

```
"lint": "npm run eslint:check && npm run prettier:check",
 "eslint:check": "eslint . --ext .json,.js,.jsx,.ts,.tsx",
```
To make sure both of the tools run on the CI.

I understand you wanted to do it on a separate file, but I think we can speed up and add it here, and merge it at once 👍